src/pages/commitqueue/CommitQueueCard.tsx

```diff
@@ -24,6 +25,7 @@ interface Props {
   author: string;
   commitTime: Date;
   patchId: string;
+  versionId: string | undefined;
```

Comment:
is `| undefined` necessary?

Reply:
nope, I had thought the field was nullable in the query for some reason